pkg/util/ovs/fake_ovs_test.go

```diff
@@ -129,47 +129,47 @@ func TestFakeFind(t *testing.T) {
 		t.Fatalf("unexpected error adding bridge: %v", err)
 	}
 
-	vethA, err := ovsif.AddPort("vethA", -1, "external-ids=sandboxID=ALPHA")
+	vethA, err := ovsif.AddPort("vethA", -1, "external_ids=sandboxID=ALPHA")
```

Comment:
@danwinship should we have a test that ensures that fake-ovs fails correctly with "external-ids" or is that pointless?